moveit_ros/moveit_servo/src/servo_parameters.cpp

```diff
@@ -77,74 +78,259 @@ ServoParameters::setParametersCallback(std::vector<rclcpp::Parameter> parameters
   return result;
 }
 
-ServoParameters::SharedConstPtr ServoParameters::makeServoParameters(const rclcpp::Node::SharedPtr& node,
-                                                                     const rclcpp::Logger& logger,
-                                                                     std::string ns, /* = "moveit_servo"*/
-                                                                     bool dynamic_parameters /* = true */)
+void ServoParameters::declare(const std::string& ns,
+                              const rclcpp::node_interfaces::NodeParametersInterface::SharedPtr& node_parameters,
+                              std::shared_ptr<ServoParameters> parameters)
 {
-  auto parameters = std::shared_ptr<ServoParameters>(new ServoParameters(logger, ns));
+  // Properties of incoming commands
```

Comment:
What does this mean?

Reply:
See https://github.com/ros-planning/moveit2/blob/b426a1c6f957522f8261f3500ee11f82199cdef6/moveit_ros/moveit_servo/src/servo_parameters.cpp#L91

I believe is saying that the parameters below are properties of incoming commands (ex. command_in_topic)

Reply:
^yep, exactly